src/v0/destinations/customerio/transform.js

```diff
@@ -322,7 +323,7 @@ const batchEvents = (successRespList) => {
   const groupEvents = [];
   // Filtering out group calls to process batching
   successRespList.forEach((resp) => {
-    if (resp.message.endpoint !== OBJECT_EVENT_ENDPOINT) {
+    if (!OBJECT_EVENT_ENDPOINTS.includes(resp.message.endpoint)) {
```

Comment:
``` suggestion
    if (resp.message.endpoint.includes("v2/batch")) {
```

Reply:
@ItsSudip there might be possibility in future where we might introduce batching for other calls as well and in that case this condition will fail